app/web_ui/src/app.css

```
@@ -1,3 +1,19 @@
+/* Inter — self-hosted variable font. License: static/fonts/OFL.txt (SIL OFL 1.1) */
+@font-face {
+  font-family: "InterVariable";
```

Stylelint (17.14.0)

[error] Expected no quotes around "InterVariable" (font-family-name-quotes)

(font-family-name-quotes)